.babelrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   ],
   plugins: [
     '@babel/plugin-transform-flow-strip-types',
+    '@babel/plugin-transform-runtime',
```

Comment:
could you remove it from `.babelrc.js` and add it like this: https://github.com/final-form/react-final-form/pull/351/files#diff-ff6e5f22a9c7e66987b19c0199636480R86 ?

Reply:
@Andarist Fixed ✅